src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -381,6 +383,11 @@ TEST_P(OVClassSetDevicePriorityConfigTest, SetConfigNoThrow) {
     ASSERT_EQ(devicePriority, configuration[ov::device::priorities.name()].as<std::string>());
 }
 
+TEST_P(OVClassSetAllowAutoBatchingConfigTest, SetConfigNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
```

Comment:
Why do you add this test? We have [similar one](https://github.com/openvinotoolkit/openvino/blob/43784e2ceccdc909456161bf950a163bb189902a/src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp#L287)

Reply:
As discussed in the https://github.com/openvinotoolkit/openvino/pull/10368#discussion_r807035896,  I updated this test case and made the list of configs be passed from the plugin(CPU/GPU). Now after clarifying the scope of this PR, I will  only cover test case of config ALLOW_AUTO_BATCHING on AUTO/MULTI using set_config()/get_config() and set_property()/get_property().